src/components/Calculator.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from 'react';
+import './calculator.css';
+import CalculatorInput from './calculatorInput';
+import calculate from '../logic/calculator';
+import Qoutes from './qoutes';
```

Comment:
- [ ] The Quotes component should not be imported in the Calculator component. They should be imported in the App component. Kindly move this import.